level1/outputs.tf

```diff
@@ -0,0 +1,15 @@
+output "public_subnet_id" {
+    value = aws_subnet.public[*].id
+}
+
+output "private_subnet_id" {
+    value = aws_subnet.private[*].id
+}
+
+output "vpc_id" {
+    value = aws_vpc.main.id
+}
+
+output "vpc_cidr" {
+    value = var.vpc_cidr
+}
```

Comment:
could you please, add newline at the end of file.

Reply:
done